### PR TITLE
Adding travis-ci badge to readme

### PR DESCRIPTION
**Pull request by tldavies**

### CODE REVIEW

**Comment by tldavies** [comment]
@DavidjohnBlodgett @RackHD/corecommitters @benbp

**Comment by DavidjohnBlodgett** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
-on-imagebuilder
+on-imagebuilder [![Build Status](https://travis-ci.org/RackHD/on-web-ui.svg?branch=master)](https://travis-ci.org/RackHD/on-web-ui)
```
Comment (yyscamper) [inline README.md]: Why the repo name is "on-web-ui"? shouldn't be on-imagebuilder?

Comment (heckj) [inline README.md]: good catch felix - @tldavies looks like a copy/paste error :-)

**Comment by yyscamper** [comment]
@tldavies , please fix the repo name.

**Comment by benbp** [comment]
@tldavies bump

**Comment by benbp** [comment]
Merging before build complete since it's just a README update.

**Pull request merged**